compiler/ballerina-parser/src/test/resources/declarations/record-type-def/record_type_def_source_19.bal

```diff
@@ -1,6 +1,4 @@
 type Foo record {
-    readonly T a;
     readonly a;
     readonly a = v;
-    readonly & T a = v;
```

Comment:
This is a valid code right?

Reply:
yes aiye. Since I have added 
```
type Foo record {
    int[] & readonly x = [1, 2];
    readonly & int[] y = [1, 2];
};
```
in a separate test case I removed it from there.

Reply:
ah ok